dashboard/test/controllers/courses_controller_test.rb

```diff
@@ -7,35 +7,124 @@ class CoursesControllerTest < ActionController::TestCase
     @teacher = create :teacher
     sign_in @teacher
 
+    @levelbuilder = create :user
```

Comment:
This can be more succinctly written as [`@levelbuilder = create :levelbuilder`](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/factories.rb#L51-L56).